tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -669,6 +669,310 @@ def test_callf_stack_height_limit_exceeded(eof_test, callee_outputs):
+            Section.Code(
+                code=Op.PUSH1[1] * stack_height + Op.CALLF[1] + Op.POP * stack_height + Op.RETF,
+                code_outputs=0,
+                max_stack_height=stack_height,
+            ),
+        ],
+    )
+    stack_overflow = stack_height > MAX_RUNTIME_OPERAND_STACK_HEIGHT // 2
+    eof_test(
+        container=container,
+        expect_exception=EOFException.STACK_OVERFLOW if stack_overflow else None,
+    )
+
+
+@pytest.mark.parametrize("stack_height", [1, 2])
+def test_callf_stack_overflow_after_callf(eof_test: EOFTestFiller, stack_height: int):
+    """Test CALLF intruction calling next function causing stack overflow at validation time."""
+    container = Container(
+        sections=[
+            Section.Code(code=Op.CALLF[1] + Op.STOP),
+            Section.Code(
+                code=Op.PUSH1[1] * 1023 + Op.CALLF[2] + Op.POP * 1023 + Op.RETF,
+                code_outputs=0,
+                max_stack_height=1023,
+            ),
+            Section.Code(
+                code=Op.PUSH0 * stack_height + Op.POP * stack_height + Op.RETF,
+                code_outputs=0,
+                max_stack_height=stack_height,
+            ),
+        ],
+    )
+    stack_overflow = 1023 + stack_height > MAX_RUNTIME_OPERAND_STACK_HEIGHT
+    eof_test(
+        container=container,
+        expect_exception=EOFException.STACK_OVERFLOW if stack_overflow else None,
+    )
+
+
+@pytest.mark.parametrize("stack_height", [512, 514, 515])
+def test_callf_stack_overflow_variable_stack(eof_test: EOFTestFiller, stack_height: int):
+    """Test CALLF instruction causing stack overflow."""
+    container = Container(
+        sections=[
+            Section.Code(
+                code=Op.PUSH0
+                + Op.PUSH1[0]
+                + Op.RJUMPI[2]
+                + Op.PUSH0 * 2
+                + Op.PUSH1[1] * 509
+                + Op.CALLF[1]
+                + Op.STOP,
+                max_stack_height=512,
+            ),
+            Section.Code(
+                code=Op.PUSH1[1] * stack_height + Op.POP * stack_height + Op.RETF,
+                code_outputs=0,
+                max_stack_height=stack_height,
+            ),
+        ],
+    )
+    stack_overflow = stack_height > MAX_RUNTIME_OPERAND_STACK_HEIGHT // 2
+    eof_test(
+        container=container,
+        expect_exception=EOFException.STACK_OVERFLOW if stack_overflow else None,
+    )
+
+
+@pytest.mark.parametrize("stack_height", [1, 2, 5])
+def test_callf_stack_overflow_variable_stack_2(eof_test: EOFTestFiller, stack_height: int):
+    """Test CALLF instruction causing stack overflow."""
+    container = Container(
+        name="callf_stack_overflow_variable_stack_3",
+        sections=[
+            Section.Code(
+                code=Op.PUSH0
+                + Op.PUSH1[0]
+                + Op.RJUMPI[2]
+                + Op.PUSH0 * 2
+                + Op.PUSH1[1] * 1020
+                + Op.CALLF[1]
+                + Op.STOP,
```

Comment:
```suggestion
                code=Op.RJUMPI[2](0)
                + Op.PUSH0 * (MAX_RUNTIME_OPERAND_STACK_HEIGHT - 1)
                + Op.CALLF[1]
                + Op.STOP,
```
Similar to previous comment.